code/modules/interface/multiContext/context_actions.dm

```diff
@@ -1270,3 +1269,56 @@
 		name = "Released"
 		icon_state = "released"
 		mode = PRISONER_MODE_RELEASED
+
+/datum/contextAction/omnitool
+	icon = 'icons/ui/context16x16.dmi'
```

Comment:
this should live with the omnitool imo

Reply:
You mean the whole context action? isn't keeping all the context actions together what this file is for?